superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -420,6 +420,7 @@ export const TRANSLATIONS = {
   CUSTOM_SCREENSHOT_WIDTH_TEXT: t('Screenshot width'),
   CUSTOM_SCREENSHOT_WIDTH_PLACEHOLDER_TEXT: t('Input custom width in pixels'),
   NOTIFICATION_METHOD_TEXT: t('Notification method'),
+  SUBJECT_TEXT: t('Subject name'),
```

Comment:
"name" doesn't seem right here. Subject line, or subject text, or just subject, makes more sense I think.

Reply:
I already resolve this one it is unused code ad I already remove it thanks